aries-site/src/pages/components/selectmultiple.mdx

```diff
@@ -0,0 +1,48 @@
+import { Example } from '../../layouts';
+import { SelectMultipleAnatomyExample, SelectMultipleSimpleExample } from '../../examples';
+
+The SelectMultiple component is typically used as a form element but can also be leveraged for user experiences that include searching or narrowing down data. It contains an optional search field. When within a form, the SelectMultiple will support labels, error and disabled states.
+
+<Example
+  componentName="SelectMultiple"
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/selectmultiple/SelectMultipleSimpleExample.js"
+  docs="https://v2.grommet.io/selectmultiple?theme=hpe#props"
+  figma="https://www.figma.com/file/EcLSFz5L7EwmOadmEYGshZ/SelectMultiple-Component?node-id=1%3A1026"
+  grommetSource="https://github.com/grommet/grommet/blob/master/src/js/components/SelectMultiple/SelectMultiple.js"
+>
+  <SelectMultipleSimpleExample />
+</Example>
+
+## When to use
+
+When multiple selections can be made in a list that contains 5 or more items.
+
+## When not to use
+
+- For lists that contain less than 5 options to choose from, consider using a [CheckBoxGroup](/components/checkboxgroup) instead. 
+- When only a single selection is permitted, use the [Select](/components/select) component instead.
+
+## Anatomy
+
+<Example plain >
+  <SelectMultipleAnatomyExample />
+</Example>
+
+1. **Select**: Displays current input value and control to open/close options drop
```

Comment:
```suggestion
1. **Select**: Displays current input value and control to open/close options drop.
```

Reply:
Arguably this is a statement and not a complete sentence (same as line after it.)